addons/account/models/company.py

```diff
@@ -343,6 +344,7 @@ def create_op_move_if_non_existant(self):
                 'name': _('Opening Journal Entry'),
                 'company_id': self.id,
                 'journal_id': default_journal.id,
+                'date': datetime.today().date().replace(month=1, day=1),
```

Comment:
fiscal year doesn't start the 1 junary for all company, it could be better to use the value in the accounting setting (if not use 1/1/20xx).

Reply:
I think it is better to use the day before `fiscalyear_last_day` and `fiscalyear_last_month`, if the accountant have set in config, when you generate the opening move, it set the value set by the accountant (and not 1/1/20xx by default).

Reply:
Thanks, after discussion with the relevant authorities it is also included in this PR.